javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,12 @@
+/* exported oddOrEven */
+function oddOrEven(numbers) {
+  var oddOrEven = [];
+  for (var i = 0; i < numbers.length; i++) {
+    if ((numbers[i] % 2) === 0) {
+      oddOrEven.push('even');
+    } else if ((numbers[i] % 2) === 1) {
```

Comment:
This could just be an `else` statement because you are checking for the opposite of the original conditional statement.